ingestion_tools/scripts/common/config.py

```diff
@@ -250,6 +230,7 @@ def resolve_output_path(self, key: str, obj: BaseImporter) -> str:
             "frame": "{dataset_name}/{run_name}/Frames",
             "rawtilt": "{dataset_name}/{run_name}/TiltSeries",
             "collection_metadata": "{dataset_name}/{run_name}/TiltSeries",
+            "alignment": "{dataset_name}/{run_name}/Alignments",
```

Comment:
"gain" on line 249 needs to be updated as part of this work.

Reply:
Addressed in https://github.com/chanzuckerberg/cryoet-data-portal-backend/pull/271